### PR TITLE
new data 1 added to feature1

### PR DESCRIPTION
**Pull request by rajeshrajkr4**

### CODE REVIEW

**Review by rahul37865 — CHANGES_REQUESTED** [review]
not possible

File: file1.txt [hunk 1]
```diff
@@ -1 +1 @@
-Hello test2345678922
\ No newline at end of file
+Hello test234
```
Comment (rahul37865) [inline file1.txt:1]: add 0 to this number

Comment (rajeshrajkr4) [reply file1.txt]: I added 0

**Review by rahul37865 — CHANGES_REQUESTED** [review]
can you also add 1

**Pull request merged**